### DISCUSSION TITLE
Redis 中的事件循环 · /redis-eventloop

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#269] [Announcements]
https://draveness.me/redis-eventloop

**Comment by chushu10** [comment]
请问图片里的笔记是用的什么软件呢？

**Comment by draveness** [comment]
@Choosue Paper 53

**Comment by OrdinaryYZH** [comment]
博主能够加个目录导航栏不，文章太长阅读不太方便呢

**Comment by wujunze** [comment]
👍

**Comment by chemila** [comment]
请教博主，这些图片用的什么软件？

**Comment by Damanchen** [comment]
博主总结的很棒，谢过大佬~
提点建议，，，倒数第三张图片里的aeApiPoll那里是不是画错了，文件事件触发时，会被aeApiPoll选出来，放到aeFiredEvent数组中，所以应该不用再画到那个大框框里了吧。另外，建议在第三张图中加一个表示aeFiredEvent的框框，这样会不会更好一些呢。

**Comment by junxxx** [comment]
博主，我发现您博客里的图片采用了单独的域名：img.draveness.me，跟博客的域名是分开的。我好奇的是，图片存储这块你是自己写的服务，还是采用的第三方图片存储服务啊？

**Comment by draveness** [comment]
> 博主，我发现您博客里的图片采用了单独的域名：img.draveness.me，跟博客的域名是分开的。我好奇的是，图片存储这块你是自己写的服务，还是采用的第三方图片存储服务啊？

第三方的图片存储服务
